docs/documentation/theming.md

```diff
@@ -142,9 +142,16 @@ cycle = [[:linecolor, :markercolor] => :color, :marker]
 cycle = nothing # equivalent to cycle = []
 ```
 
+Notice that cycles must be given as attributes to a plot object, not the top-level theme
+(because different plot objects can cycle different attributes, e.g., a density plot
+cannot cycle markers). This is exemplified in the following code blocks.
+
 \begin{examplefigure}{}
 ```julia
-set_theme!(palette = (color = [:red, :blue], marker = [:circle, :xcross]), Scatter = (cycle = [:color, :marker],))
+set_theme!(
```

Comment:
Could you please change all `set_theme!`s to `with_theme` blocks? It happens again and again that the global theme is modified and not reset accidentally and then some random plots somewhere else start looking wonky :) So the `with_theme` guards against that